packages/web-console/src/scenes/Editor/Monaco/questdb-sql/createSchemaCompletionProvider.ts

```diff
@@ -1,55 +1,129 @@
 import { Table } from "../../../../utils"
 import * as monaco from "monaco-editor"
-import { CompletionItemKind } from "./types"
+import { InformationSchemaColumn } from "./types"
+import { editor, IRange } from "monaco-editor"
+import { languages } from "monaco-editor"
+import IStandaloneCodeEditor = editor.IStandaloneCodeEditor
+import { findMatches, getQueryFromCursor } from "../utils"
 
-export const createSchemaCompletionProvider = (questDBTables: Table[] = []) => {
+const getColumnCompletion = (
+  column: InformationSchemaColumn,
+  range: IRange,
+) => ({
+  label: {
+    label: column.column_name,
+    detail: ` (${column.table_name})`,
+    description: column.data_type,
+  },
+  kind: languages.CompletionItemKind.Enum,
+  insertText: column.column_name,
+  sortText: column.table_name,
+  range,
+})
+
+export const createSchemaCompletionProvider = (
+  editor: IStandaloneCodeEditor,
+  tables: Table[] = [],
+  informationSchemaColumns: InformationSchemaColumn[] = [],
+) => {
   const completionProvider: monaco.languages.CompletionItemProvider = {
-    triggerCharacters: "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz \"".split(
-      "",
-    ),
+    triggerCharacters:
+      'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz\n "'.split(""),
```

Comment:
Trivial readability nit:
```suggestion
      Array.from('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz\n "'),
```